apps/prairielearn/src/lib/question.js

```diff
@@ -1862,6 +1918,8 @@ module.exports = {
         )
       );
 
+      /** @type function */
+      let renderFileAsync = util.promisify(ejs.renderFile);
```

Comment:
For some reason, type checking on this expression was failing without this explicit declaration. 

see https://github.com/DefinitelyTyped/DefinitelyTyped/issues/33340